bundle/0.1.0/manifests/nginx-ingress-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,374 @@
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - apiextensions.k8s.io
+          resources:
+          - customresourcedefinitions
+          verbs:
+          - create
+          - delete
+          - update
+          - get
+        - apiGroups:
+          - networking.k8s.io
+          resources:
+          - ingresses
+          verbs:
+          - list
+          - watch
+          - get
+        - apiGroups:
+          - networking.k8s.io
+          resources:
+          - ingresses/status
+          verbs:
+          - update
+        - apiGroups:
+          - security.openshift.io
+          resources:
+          - securitycontextconstraints
+          verbs:
+          - create
+          - update
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - networking.k8s.io
+          resources:
+          - ingressclasses
+          verbs:
+          - get
+          - create
+        serviceAccountName: nginx-ingress-operator
+      deployments:
+      - name: nginx-ingress-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: nginx-ingress-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: nginx-ingress-operator
+            spec:
+              containers:
+              - command:
+                - nginx-ingress-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: nginx-ingress-operator
+                image: nginx/nginx-ingress-operator:0.1.0
+                imagePullPolicy: IfNotPresent
+                name: nginx-ingress-operator
+                resources: {}
+              serviceAccountName: nginx-ingress-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: true
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - nginx
+  - ingress-controller
+  - ingress
+  - controller
+  - kubernetes
+  - openshift
+  links:
+  - name: Documentation for NGINX Plus
+    url: https://github.com/nginxinc/nginx-ingress-operator/tree/release-0.1.0/examples/deployment-plus-min
+  - name: Documentation for NGINX Open Source
+    url: https://github.com/nginxinc/nginx-ingress-operator/tree/release-0.1.0/examples/deployment-oss-min
+  - name: Documentation for NginxIngressController CR
+    url: https://github.com/nginxinc/nginx-ingress-operator/tree/release-0.1.0/docs/nginx-ingress-controller.md
```

Comment:
Note: Previous releases referenced docs/examples on `master` this release I changed it to reference the release branch which will be cut once this PR is merged.